content/applications/finance/documents.rst

```diff
@@ -44,33 +44,54 @@ Workspaces
 
 Workspaces are hierarchical folders having their own set of tags and actions. Default workspaces
 exist, but you can create your own by going to :menuselection:`Documents --> Configuration -->
-Workspaces` and clicking on :guilabel:`Create`.
+Workspaces` and clicking on :guilabel:`New`. A new page pops up:
 
-.. note::
-   :guilabel:`Workspaces` and :guilabel:`Sub-workspaces` can be created, edited, or deleted by
-   clicking on the gear icon :guilabel:`⚙` on the left menu.
-
-.. image:: documents/sub-workspaces-creation.png
-   :align: center
-   :alt: Create sub-workspaces from the left menu
+- :guilabel:`Name`: add the name of your workspace;
+- :guilabel:`Parent Workspace`: if the workspace belongs to a parent workspace (and is then a
+  sub-workspace), select it from the list;
+- :guilabel:`Company`: if your workspace is only available to the selected company;
 
 Tags
-====
+----
 
 Tags are used within workspaces to add a level of differentiation between documents. They are
 organized per category and filters can be used to sort them.
```

Comment:
Tags are used within workspaces to add a level of differentiation between documents. They are
organized per category, and filters can be used to sort them.